Product

```diff
@@ -0,0 +1,34 @@
+class Product: 
+    def __init__(self, sale_price, number_products, purchase_price, name, brand, image, barcode, instructions, 
+        size):
+        self.sale_price = sale_price # needs to be an integer
+        self.number_products = number_products # needs to be an integer
+        self.purchase_price = purchase_price # needs to be an integer
+        self.name = name # needs to be a string
+        self.brand = brand # needs to be a string 
+        self.image = image # needs to be a string
+        self.barcode = barcode # needs to be boolean 
+        self.instructions = instructions # needs to be a string 
+        self.size = size # needs to be an integer 
+
+
+    def design(): # we will need certain sizes, materials and colors
```

Comment:
I don't think this is a characteristic of a product.

Reply:
I removed unnecessary attributes to maintain a clean and understandable framework for my program.